Rakefile

```diff
@@ -280,6 +293,25 @@ def app_version
   end
 end
 
+def ios_build_config
+  signing_style = IOS_SIGNING_STYLE
+  team_id = ENV['IOS_DEVELOPMENT_TEAM_ID']
+
+  if(environment === 'production')
+    provisioning_profile = ENV['PROVISIONING_PROFILE_PROD']
```

Comment:
@abulsayyad123 can you please use `PROVISIONING_PROFILE_ADMIN_PROD` instead of `PROVISIONING_PROFILE_PROD` and same for Staging that way on local we will be able to build staging and prod build on local for all the apps.

Reply:
@shivanibhanwal , sure will add these changes.